andela/AndelaEatsUI
# #160536352 Implement admin create meals

**What does the PR do?**
- Add functionality to enable `admin` add meals

**Description of Task to be completed?**
- Update add meal component to connect to store
- Define action creators to update the store
- Make a post request to add a meal
- Handle response from post request

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
- Change directory into AndelaEatsUI and checkout to this branch `ft-admin-create-meal-160536352`
- Run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, select `Meals`, and click on `Add meal`
- Complete the form and click the `Add meal` button
- run `npm test` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#160536352](https://www.pivotaltracker.com/story/show/160536352)

**Screenshots**

<img width="1440" alt="screen shot 2018-09-24 at 1 55 03 am" src="https://user-images.githubusercontent.com/32123313/45934922-ec982f00-bf9c-11e8-8fa4-fc299fcb7291.png">
<img width="1440" alt="screen shot 2018-09-24 at 1 55 05 am" src="https://user-images.githubusercontent.com/32123313/45934923-ec982f00-bf9c-11e8-926c-393903d9b080.png">
<img width="1440" alt="screen shot 2018-09-24 at 1 55 08 am" src="https://user-images.githubusercontent.com/32123313/45934924-ec982f00-bf9c-11e8-9097-e7187b7a0865.png">
